actions/test-repository/action.yml

```diff
@@ -0,0 +1,42 @@
+name: 'Test TUF-on-CI repository'
+description: 'Test a published TUF-on-CI repository with a client'
+
+inputs:
+  metadata_url:
+    description: 'base metadata URL the client should use'
+    default: ''
+  artifact_url:
+    description: 'Base artifact URL the client should use'
+    default: ''
```

Comment:
How about an optional third parameter that would be a list of targets to download as well? I would think about making it simple, i.e only test for existence, not for verifying content (i.e the message digest).

Reply:
Yeah I agree it makes sense: left it out of this version to keep it short but I could do your simplified suggestion...

Reply:
I've added an optional `expected_artifact` argument. I don't know if it's actually useful for most repos but it'll likely be useful for root-signing and it's not much more complicated.